internal/cmd/finding/finding.go

```diff
@@ -86,50 +90,131 @@ func newWithOptions(opts *options) *cobra.Command {
 		cmdutils.AddProjectDirFlag,
 		cmdutils.AddInteractiveFlag,
 		cmdutils.AddServerFlag,
+		cmdutils.AddProjectFlag,
 	)
 
 	return cmd
 }
 
 func (cmd *findingCmd) run(args []string) error {
-	errorDetails, err := auth.TryGetErrorDetails(cmd.opts.Server)
+	errorDetails, token, err := auth.TryGetErrorDetailsAndToken(cmd.opts.Server)
+	if err != nil {
+		var connErr *api.ConnectionError
+		if !errors.As(err, &connErr) {
+			return err
+		} else {
+			log.Debugf("Connection error: %v", connErr)
+		}
+	}
```

Comment:
The `api.ConnectionError` is already handled by auth.TryGetErrorDetailsAndToken
```suggestion
	errorDetails, token, err := auth.TryGetErrorDetailsAndToken(cmd.opts.Server)
	if err != nil {
		return err
	}
```

Reply:
Fixed, thx!